drivers/bt_ll_softdevice/flash/flash_ll_softdevice.c

```diff
@@ -17,8 +17,19 @@
 #include "sdc_soc.h"
 #include "multithreading_lock.h"
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, nordic_nrf51_flash_controller), okay)
```

Comment:
The SoftDevice controller is only supported on nrf52 and nrf53. Other devices may be removed from here

Reply:
fixed